src/player.js

```diff
@@ -479,6 +487,7 @@ export default class Player extends FakeEventTarget {
       Utils.Object.mergeDeep(this._config, config);
     }
     this._applyTextTrackConfig(config);
+    this._applyABRRestriction(config);
```

Comment:
so there is no api `applyABRRestriction` (as in the PR desc) ? only by configure?

Reply:
Yes, discuss with Oren a long time ago and we decided that it much more make sense to keep the configuration in one code path.